doc/sedcli.8

```diff
@@ -91,6 +91,9 @@ MBR Done update to TRUE will only take effect when MBR Shadow is enabled.
 Write data into MBR shadow region(MBR table).
 MBR shadow should be enabled for the PBA to take effect.
 
+.IP "\fB\-B -d <device>\fR or \fB\-\-block_sid --device <device> --hwreset {1|0}\fR"
```

Comment:
--hwreset option is missing in the long short form. Should be:
.IP "\fB\-B -d <device>-r {1|0} \fR or \fB\-\-block_sid --device <device> --hwreset {1|0}\fR"

Reply:
thanks